CHANGELOG.rst

```diff
@@ -10,7 +10,7 @@ v3.1.0
 Release Summary
 ---------------
 
-This release includes bug fixes and new feature for the ``terraform_state`` inventory plugin.
+This release includes bug fixes and new feature for the ``terraform_state`` inventory plugin and ``terraform`` module plugin.
```

Comment:
Were these added manually? If so, please remove these and only keep the changelog fragment entry.